examples/fluids/src/misc.c

```diff
@@ -62,11 +62,12 @@ PetscErrorCode SetBCsFromICs(DM dm, Vec Q, Vec Q_loc) {
   PetscCall(PetscObjectComposeFunction((PetscObject)dm, "DMPlexInsertBoundaryValues_C", DMPlexInsertBoundaryValues_FromICs));
 
   PetscCall(DMGetNamedLocalVector(dm, "boundary mask", &boundary_mask));
-  PetscCall(DMGetGlobalVector(dm, &Q));
+  PetscCall(DMGetGlobalVector(dm, &ones));
   PetscCall(VecZeroEntries(boundary_mask));
-  PetscCall(VecSet(Q, 1.0));
-  PetscCall(DMGlobalToLocal(dm, Q, INSERT_VALUES, boundary_mask));
+  PetscCall(VecSet(ones, 1.0));
+  PetscCall(DMGlobalToLocal(dm, ones, INSERT_VALUES, boundary_mask));
   PetscCall(DMRestoreNamedLocalVector(dm, "boundary mask", &boundary_mask));
+  PetscCall(DMRestoreGlobalVector(dm, &ones));
```

Comment:
Purely stylistic, but you could declare it closer.
```c
{
  Vec ones;
  PetscCall(DMGetGlobalVector(dm, &ones));
  PetscCall(VecZeroEntries(boundary_mask));
  PetscCall(VecSet(ones, 1.0));
  PetscCall(DMGlobalToLocal(dm, ones, INSERT_VALUES, boundary_mask));
  PetscCall(DMRestoreNamedLocalVector(dm, "boundary mask", &boundary_mask));
  PetscCall(DMRestoreGlobalVector(dm, &ones));
}
```

Reply:
Yep, I thought about doing that too, but didn't execute for whatever reason.